.github/workflows/smoke.yml

```diff
@@ -28,6 +28,9 @@ jobs:
           sh -c "$(curl --location https://raw.githubusercontent.com/F1bonacc1/process-compose/main/scripts/get-pc.sh)" --
           -d -b ~/bin
 
+      - name: Install grpcurl
+        run: ./deployments/scripts/install-grpcurl
+
```

Comment:
@cratelyn N.B. I'd much rather use something like nix flakes or devbox to manage these dependencies. Not going down that rabbit hole now, but flagging that it'd be less work overall to track tooling deps in a holistic way, cf. #4264.